spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -149,6 +232,12 @@ def problem_message(attribute, expected, actual)
       "#{attribute} doesn't match, expected: #{expected.inspect}, actual: #{actual.inspect}"
     end
 
+    def unpublishing_problem_message(publishing_api_type, expected)
+      "unpublishing type not expected, " +
+        "expected: #{publishing_api_type.inspect}, " +
+        "actual: #{expected.inspect}"
+    end
```

Comment:
We don't need this anymore.

Reply:
I will need to adapt the `problem_message` method in order to get rid of this.

Reply:
Sorted here: https://github.com/alphagov/content-publisher/pull/1883/commits/1390eaf1624423299dbccf0f9461f3f40033f261